src/main/java/com/ase/restservice/controller/AccountController.java

```diff
@@ -96,4 +101,11 @@ public Float getAccountPortfolioValue(@PathVariable(value = "accountId") final S
       throws ResourceNotFoundException {
     return assetService.getAccountPortfolioValue(accountId);
   }
+  
+  @Operation(summary = "Get all transactions (buy/sell orders) for an account given accountId")
+  @GetMapping("/accounts/{accountId}/transactions")
+  public List<Transaction> getAccountTransactions(
```

Comment:
Can we name this endpoint listAccountTransactions? My reasoning for this is to use the 'get' keyword for things like getBalance and getStock that returns a single entity. Since this function returns a list of transactions, a better action verb would be 'list'. This also makes me realize that we should also rename getAllAssets and similar endpoints to listAssets etc.

Reply:
While we are on it, can we also add a listAccounts endpoint that returns all accounts?